cases/management/commands/extract_narrative_defendants.py

```
@@ -0,0 +1,609 @@
+            try:
+                return _call_llm_for_case(
+                    section_label=section_label,
+                    serial=serial,
+                    case_text=case_text,
+                    model=model,
+                    base_url=base_url,
+                    api_key=api_key,
+                    timeout=timeout,
+                    use_anthropic=options.get("use_anthropic", False),
+                )
+            except Exception as e:
```

Ruff (0.15.15)

[warning] Do not catch blind exception: `Exception`

(BLE001)